vexim/adminuserchangesubmit.php

```diff
@@ -30,10 +30,15 @@
   }
   if (isset($_POST['on_forward'])) {
     $_POST['on_forward'] = 1;
-    if(!filter_var($_POST['forward'], FILTER_VALIDATE_EMAIL)) {
-      header ("Location: adminuser.php?invalidforward=".htmlentities($_POST['forward']));
-      die;
+    $forwardto=explode(",",$_POST['forward']);
```

Comment:
alternatively the contents of `$_POST['forwardmenu']` could be appended to `$_POST['forward']`, but this would mean changing strings in the UI as well.